config/routes.rb

```diff
@@ -3,4 +3,14 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/dishes/:dish_id", to: "dishes#show"
+
+  post "/dishes/:dish_id/ingredients", to: "dish_ingredients#create"
+
+  get "/chefs/:chef_id", to: "chefs#show"
+
+  get "/chefs/:chef_id/ingredients", to: "chef_ingredients#index"
+  
+  delete "/dishes/:dish_id", to: "dish_ingredients#destroy"
```

Comment:
I would just update this path to "/dishes/:dish_id/ingredients/:ingredient_id" to include both  the resources and their IDs, since you are deleting the relationship between them rather than deleting one of the actual resources. This also helps prevent the need to pass the ingredient ID as a parameter with the button.